core/modules/color/color.module

```diff
@@ -411,7 +411,9 @@ function color_get_palette($theme, $default = FALSE) {
   // Fetch and expand default palette.
   $info = color_get_info($theme);
   $palette = $info['schemes']['default']['colors'];
-
+  foreach ($palette as $k => $v) {
+  $palette[$k] = backdrop_strtolower($v);
```

Comment:
Can we please give some meaningful names to these variables? Something like this perhaps?:
```suggestion
  foreach ($palette as $item => $color) {
    $palette[$item] = backdrop_strtolower($color);
```

Also, according to our coding standards, the lines should be indented by 2 spaces. See https://docs.backdropcms.org/php-standards#indenting

Reply:
@ericfoy another thought here would be to use `array_map()` instead of the `foreach`. Something like this:
$palette = array_map('strtolower', $palette);

Reply:
The `array_map()` callback can also be `'backdrop_strtolower'`.

Reply:
1. Meaningful names: Yes, I agree, and normally I would have done exactly that. In this case, however, I had simply lifted a snippet from line 664 in this same file. This explains my use of the cryptic $k and $v... I actually was following an establish convention ;-) But I'm in favor of the proposed change.
2. The indentation: My bad. Sorry about that.
3. array_map vs. foreach: Personally, I like foreach here because (to me) it is a clearly procedural-looking, straight-forward manipulation of the data at hand, whereas (probably due to my PHP inexperience) the array_map is less obvious to me. This might be simply because this kind of construct (passing a function name as a parameter), though obviously amazingly powerful, is somewhat new to me. That being said, I looked into the speed comparisons of the two functions, and there appears to be no real advantage either way. I would therefore argue in favor of foreach, also just for the sake of symmetry with the other sections of code in this file.

So... how does this work? Should I re-edit, and resubmit? Or do you guys just take it from here? Please advise.

P.S: I like this array_map thing, and I can already tell you: I'm gonna use that sucker, and I already know several places where...

Reply:
Also... Should I click Resolve?
I'm still learning the workflow, so I don't know what is appropriate, and what might be an egregious breach of collectively understood protocol...;-)